.changeset/pre.json

```diff
@@ -52,6 +52,8 @@
     "@backstage/plugin-azure-devops": "0.1.24",
     "@backstage/plugin-azure-devops-backend": "0.3.14",
     "@backstage/plugin-azure-devops-common": "0.2.4",
+    "@backstage/plugin-azure-functions": "0.1.0",
```

Comment:
I don't believe you need to make changes in this file as it is managed by automation.

You'll want to run `yarn changeset` to generate a proper changeset file for this

Reply:
I thought so as well, thanks for clearing that up and supplying the command I need to run - I must have missed that in the documentation.